### PR TITLE
Importing sprites and movement realization

### PR DESCRIPTION
**Pull request by ShovelD**

### CODE REVIEW

File: Tank Survivors/Assets/JoystickPackage/Scripts/Editor/DynamicJoystickEditor.cs
Comment (Kiuh) [inline Tank Survivors/Assets/JoystickPackage/Scripts/Editor/DynamicJoystickEditor.cs:1]: For the editor scripts also add namespaces and formatting

File: Tank Survivors/Assets/Scripts/Tank/Moving.cs [hunk 1]
```diff
@@ -7,10 +7,33 @@ public class Moving : MonoBehaviour
     {
         [SerializeField]
         private TankImpl tank;
+        private float movementSpeed;
+        private Rigidbody2D tankRigidBody;
+        public Vector2 MovementDirection { get; set; }
```
[the remark this answers is not in the document]
Comment (Kiuh) [reply Tank Survivors/Assets/Scripts/Tank/Moving.cs:12]: and if you want to interact with it from other script, make property for created field

**Review by Kiuh — CHANGES_REQUESTED** [review]

File: Tank Survivors/Assets/Scripts/Tank/Moving.cs [hunk 2]
```diff
@@ -7,10 +7,33 @@ public class Moving : MonoBehaviour
...
 
-        public void Move()
+        public void Start()
         {
-            // TODO: Move logics
+            tankRigidBody = tank.GetComponent<Rigidbody2D>();
```
[the remark this answers is not in the document]
Comment (ShovelD) [reply Tank Survivors/Assets/Scripts/Tank/Moving.cs]: fixed

File: Tank Survivors/Assets/JoystickPackage/Scripts/Joysticks/DynamicJoystick.cs
[the remark this answers is not in the document]
Comment (ShovelD) [reply Tank Survivors/Assets/JoystickPackage/Scripts/Joysticks/DynamicJoystick.cs:1]: all scripts for joyticks now have namespace and they was formated with formatter

**Review by Kiuh — APPROVED** [review]
Apply formatting to joystick package and you can merge

**Pull request merged**